pkgs/development/libraries/mesa/default.nix

```diff
@@ -204,7 +173,7 @@ self = stdenv.mkDerivation {
     "out" "dev" "drivers"
   ] ++ lib.optionals enableOSMesa [
     "osmesa"
-  ] ++ lib.optionals stdenv.isLinux [
+  ] ++ [
     "driversdev"
```

Comment:
I'd move this up to the unilateral section so it looks neater, even thought it causes a rebuild.

Reply:
I'll do more cleanups in a follow-up PR to staging.